style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  min-height: 100vh;
```

Comment:
Avoid using `100vh` for the body, it's a bit of a hack. Use percentages if possible.